src/components/asset-list/RecyclerAssetList2/core/RowRenderer.tsx

```diff
@@ -88,6 +88,7 @@ function rowRenderer(
         <WrappedTokenFamilyHeader
           image={image}
           name={name}
+          testID={`token_family_header.${name}`}
```

Comment:
Useful for testing

Reply:
can we use 
`token-family-header-${name}`?
I think this more common pattern here?